File: src/model/Customer.java
package model;

import java.time.LocalTime;

/**
 * Customer class used for customer information.
 * 
 */

public class Customer {
    private long id;
    private String streetName;
    private String streetNo;
    private int zipcode;
    private String city;
    private LocalTime timeOfDelivery;

    public Customer(long id, String streetName, String streetNo, int zipcode, String city, LocalTime timeOfDelivery) {
        this.id = id;
        this.streetName = streetName;
        this.streetNo = streetNo;
        this.zipcode = zipcode;
        this.city = city;
        this.timeOfDelivery = timeOfDelivery;
    }

    public long getID() {
        return id;
    }


    public LocalTime getTimeOfDelivery() {
        return timeOfDelivery;
    }

    /**
     * @param timeOfDelivery the timeOfDelivery to set
     */
    public void setTimeOfDelivery(LocalTime timeOfDelivery) {
        this.timeOfDelivery = timeOfDelivery;
    }
}

File: Model/Customer.cs
using System;
using System.Runtime.Serialization;

namespace Model
{
    [DataContract()]
    public class Customer
    {
        [DataMember()]
        public long ID { get; private set; }
        [DataMember()]
        public String StreetName { get; private set; }
        [DataMember()]
        public String StreetNo { get; private set; }
        [DataMember()]
        public int Zipcode { get; private set; }
        [DataMember()]
        public String City { get; private set; }
        [DataMember()]
        public TimeSpan TimeOfDelivery { get; private set; }

        public Customer(long ID, String StreetName, String StreetNo, int Zipcode, String City, TimeSpan TimeOfDelivery)
        {
            this.ID = ID;
            this.StreetName = StreetName;
            this.StreetNo = StreetNo;
            this.Zipcode = Zipcode;
            this.City = City;
            this.TimeOfDelivery = TimeOfDelivery;
        }
    }
}
